docs/samples/drift-detection/alibi-detect/cifar10/README.md

```diff
@@ -6,6 +6,3 @@ The architecture used is shown below and links the payload logging available wit
 
 ![Architecture](architecture.png)
 
-## Dependencies
-
-You will need a version of Knative that support V1 protocol of CloudEvents. Suggested **KNative Eventing >=0.14.2**
```

Comment:
What's the minimal knative eventing requirement now?

Reply:
I tested on GKE and Kind with Knative 0.18 and Istio 1.7.3. Is there a minimum version of Knative eventing we should support?

Reply:
I think the version with eventing v1 crd ?

Reply:
The above is required for the example notebooks to work so I think is a valid base.

Reply:
Serving version is min 0.14.3, and eventing is 0.14.2

Reply:
We should enable this line....

Reply:
the examples uses the eventing v1 version, I think 0.14 is still v1alpha1..

Reply:
eventing v1 is introduced in 0.16 https://github.com/knative/eventing/tree/release-0.16/pkg/apis/eventing/v1

Reply:
Updated to 0.18